packages/paste-core/components/select/src/types.ts

```diff
@@ -1 +1,5 @@
+import type {BoxElementProps} from '@twilio-paste/box';
```

Comment:
this is cool, should this be our preferred way of adding Element into our component types?

Reply:
Yeah I've seen this done a few places in slightly different ways. Makes me wonder if it's worth creating a type in the box component package that is just `Element`

`export type Element = BoxElementProps['element'];`

Functionally it's the same as declaring this in each file, or skipping the declaration and just using the definition, but I think it might provide a layer of consistency and clarity. @SiTaggart @TheSisb Curious what you two think. 👀